build_library/vm_image_util.sh

```diff
@@ -945,9 +964,17 @@ EOF
 }
 
 _write_ova_bundle() {
-    vmdk-convert ${VM_DST_IMG} ${VM_TMP_DIR}/vm.vmdk
```

Comment:
You are dropping this call to vmdk-convert. To fix that switch the disk image type from vmdk_scsi to vmdk_stream or whatever it is